src/spring-cloud/HISTORY.md

```diff
@@ -1,5 +1,10 @@
 Release History
 ===============
+2.1.3
+-----
+* Add a parameter '--assign-endpoint' into 'az spring-cloud app create' and 'az spring-cloud app update'
+* Deprecate the parameter '--is-public' in 'az spring-cloud app create' and 'az spring-cloud app update'
+
```

Comment:
`2.1.2` is not released yet as previous PR does not update the version in `setup.py`. Is it OK to put the release note in `2.1.2` and release with previous feature together as `2.1.2`?